packages/subscription-manager/src/components/UserSettings/UserSettings.tsx

```diff
@@ -0,0 +1,19 @@
+type SubscriptionUserSettings = Partial< {
+	subscription_delivery_mail_option: 'html' | 'text';
+	subscription_delivery_day: number; // 0-6, 0 is Sunday
+	subscription_delivery_hour: number; // 0-23, 0 is midnight
+	subscription_delivery_email_blocked: boolean;
+} >;
+
+type UserSettingsProps = {
+	value?: SubscriptionUserSettings;
```

Comment:
Could this ever be null? Since this is coming from Redux, the initialState (before data fetch) could be an empty object - this would satisfy the Partial<> (SubscriptionUserSettings).

Reply:
I believe we may not use Redux for the external portal, and we will use `react-query` only. With the current requirements, there is no need to store it in global UI state.

Reply:
Nevertheless, we can make it required and pass an empty object by default. I have no objections to that.

Reply:
Ok, in that case we wouldn't need to use @automattic/data-stores I think. We could just create hooks that wrap useQuery. 

Referencing this ticket: https://github.com/Automattic/wp-calypso/issues/74422